src/improvements/tasks/sep/011-opcm-upgrade-prestate-v300-base/VALIDATION.md

```diff
@@ -0,0 +1,163 @@
+# Validation
+
+This document can be used to validate the inputs and result of the execution of the upgrade transaction which you are
+signing.
+
+The steps are:
+
+1. [Validate the Domain and Message Hashes](#expected-domain-and-message-hashes)
+2. [Verifying the transaction input](#understanding-task-calldata)
+3. [Verifying the state changes](#state-changes)
+
+## Expected Domain and Message Hashes
+
+First, we need to validate the domain and message hashes. These values should match both the values on your ledger and
+the values printed to the terminal when you run the task.
+
+> [!CAUTION]
+>
+> Before signing, ensure the below hashes match what is on your ledger.
+>
+> ### Nested Safe 1: `0x6AF0674791925f767060Dd52f7fB20984E8639d8`
+>
+> - Domain Hash: `0x6f25427e79742a1eb82c103e2bf43c85fc59509274ec258ad6ed841c4a0048aa`
+> - Message Hash: `0x4cdac20368824e0bf4725fa51adbab279fb9e378750085a2255937b69e4b7c06`
+>
+> ### Nested Safe 2: `0x646132A1667ca7aD00d36616AFBA1A28116C770A`
+>
+> - Domain Hash: `0x1d3f2566fd7b1bf017258b03d4d4d435d326d9cb051d5b7993d7c65e7ec78d0e`
+> - Message Hash: `0x4cdac20368824e0bf4725fa51adbab279fb9e378750085a2255937b69e4b7c06`
+
+## Understanding Task Calldata
+
+This document provides a detailed analysis of the final calldata executed on-chain for the OPCM upgrade to v2.0.0.
+
+By reconstructing the calldata, we can confirm that the execution precisely implements the approved upgrade plan with no unexpected modifications or side effects.
+
+### Inputs to `opcm.upgrade()`
+
+For each chain being upgrade, the `opcm.upgrade()` function is called with a tuple of three elements:
```

Comment:
There are still references to the upgrade function in this file. In the OP,Ink VALIDATION file I renamed all of these.